bin/suitcss

```diff
@@ -1,17 +1,19 @@
 #!/usr/bin/env node
 
-var fs = require('fs-extra');
-var assign = require('object-assign-deep');
-var logger = require('./logger');
-var suitcss = require('..');
-var path = require('path');
-var program = require('commander');
-var read = require('read-file-stdin');
-var chokidar = require('chokidar');
-
-var exists = fs.existsSync;
-var resolve = path.resolve;
-var writeFileSync = fs.outputFileSync;
+'use strict';
```

Comment:
Yeah, I get an error saying to use `const` and `let` it has to be in strict mode.